package.json

```diff
@@ -57,18 +56,23 @@
       "git add"
     ]
   },
+  "np": {
+    "branch": "release"
+  },
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "build": "rimraf dist && tsc",
-    "build:docs": "rimraf docs && typedoc --out docs --exclude \"**/*.spec.ts\" --mode file --readme README.md && ./bin/sanitize-doc-refs.sh",
+    "build:docs": "rimraf docs && typedoc src/index.ts --out docs --exclude \"**/*.spec.ts\" --readme README.md && ./bin/sanitize-doc-refs.sh",
     "deploy:docs": "gh-pages -d docs",
     "lint": "tslint -p ./tsconfig.lint.json -t stylish",
     "prettier:ts": "prettier --parser typescript --single-quote true --trailing-comma es5 --write 'src/**/*.ts'",
     "prettier:ci": "prettier --parser typescript --single-quote true --trailing-comma es5 --list-different 'src/**/*.ts'",
     "prettier:json": "prettier --parser json --write '**/*.json'",
-    "release": "yarn build && np",
+    "release": "yarn build && np && yarn open:github-releases && yarn open:npm-releases",
+    "open:github-releases": "open https://github.com/maxmind/GeoIP2-node/releases",
```

Comment:
1. `open` doesn't work on my machine
2. I'm not a fan of opening more browser windows; np already opens one for the github release notes